podcastfy/config.yaml

```diff
@@ -6,13 +6,17 @@ text_to_speech:
   elevenlabs:
     default_voices:
       question: "Chris"
-      answer: "BrittneyHart"
+      answer: "River"
```

Comment:
Let's use "Jessica" here. Already published the update in the most recent release v0.2.1